docling-server/src/main.py

```
@@ -0,0 +1,108 @@
+        result = await asyncio.get_running_loop().run_in_executor(
+            None,
+            process_document,
+            tmp_path,
+            doc_id,
+            app.state.doc_converter,
+            app.state.chunker,
+            app.state.vlm_config,
+        )
+
+        # Debug: save result to file
+        debug_path = f"/tmp/debug_result_{doc_id}.json"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/debug_result_"

(S108)